src/main/resources/templates/task-form.html

```diff
@@ -17,7 +17,7 @@ <h4>Task Manager</h4>									<!-- need to find path and put it in line below --
 List of Tasks</a>
 
 <!-- Invisible form as needed to update -->
-	<input type="hidden" th:field"">
+	<input type="hidden" th:field="">
```

Comment:
Some small issues with html and css - color hex codes need to be prefixed with a #, and html attributes need a =.